<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CassignReceiptDlg</class>
 <widget class="QDialog" name="CassignReceiptDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="m_autoFillBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Fill</string>
   </property>
  </widget>
  <widget class="QTableView" name="m_transTable">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>581</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="m_availableReceiptTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>231</width>
     <height>391</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Select Receipt</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="m_clearBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>69</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelBtn">
   <property name="geometry">
    <rect>
     <x>661</x>
     <y>490</y>
     <width>69</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_okBtn">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>490</y>
     <width>69</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="m_startDate">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>45</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d MMM yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_startDateLbl">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_endDateLbl">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>11</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>End Date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="m_endDate">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>45</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d MMM yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="m_accountComboBox">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>49</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_departmentComboBox">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>16</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_sameReceiptAccount">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use same receipt account</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
